web/themes/custom/recipe_theme/src/components/navbar-brand.twig

```diff
@@ -0,0 +1,38 @@
+{#
+/**
+ * @file
+ * Template for Navbar brand component.
+ *
+ * Available config:
+ * - text: the text to use for the brand.
+ * - path: the path for the link. Leave blank for no link.
+ * - image:
+ *   - src: The source of the image
+ *   - width: The width of the image
+ *   - height: The height of the image
+ *   - alt: The alternative text of the image
+ * - utility_classes: An array of utility classes.
+ */
+#}
+{% import _self as navbar_brand %}
+{% set utility_classes = utility_classes|join(' ') %}
+
+{% if path %}
+  <a href="{{ path }}" class="navbar-brand d-flex align-items-center {{ utility_classes }}" aria-label="{{ text }}">
+    {% if image %}
+      {{ navbar_brand.image(image, width, height, alt) }}
+    {% endif %}
+    {{ text }}
+  </a>
+{% else %}
+  <span class="navbar-brand h1 mb-0 {{ utility_classes }}" aria-label="{{ text }}">
+    {% if image %}
+      {{ navbar_brand.image(image, alt) }}
+    {% endif %}
+    {{ text }}
+  </span>
+{% endif %}
+
+{% macro image(src, width, height, alt) %}
+  <img src="{{ logo.png }}" width="{{ width|default(30) }}" height="{{ height|default('auto') }}" alt="{{ alt|default('') }}" class="mr-2" />
```

Comment:
Hmm I think you shouldn't have done this. Didn't we have a very lengthy discussion before how to change the logo for the site? You can specify in the theme's info file what logo to use and I think you already did that.

I think this file is unnecessary.

Reply:
Yes, I thought it would appear somewhere else, it was a first attempt for the UI settings, I'll delete this file now.